sources/yaml/src/test/resources/org.eclipse.microprofile.config.spi.ConfigSourceProvider

```diff
@@ -0,0 +1 @@
+io.smallrye.config.source.yaml.YamlConfigSourceProvider
```

Comment:
Should this be in src/main/resources so that users only need to add the dependency to get the ConfigSource?

Reply:
Ah, yes that was the intent.  Will fix it.

Reply:
Sorry, totally missed that it should also be in `/META-INF/services` too

Reply:
Oops!